dev-libs/vc/vc-1.4.0.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="SIMD Vector Class Library for C++"
+DESCRIPTION="A library to ease explicit vectorization of C++ code."
+HOMEPAGE="https://github.com/VcDevel/Vc"
+COPYRIGHT="2006-2016 Arseny Kapoulkine"
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/VcDevel/Vc/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="449ce0a806ebb53cc99699ef223351ec770c76e819960b5849d162f9e8506d15"
+SOURCE_DIR="Vc-$portVersion"
```

Comment:
```suggestion
SOURCE_FILENAME="vc-$portVersion.tar.gz"
SOURCE_DIR="Vc-$portVersion"
```

Reply:
It provides only a static lib. Do i have to split it still into a devel package?

Reply:
It depends. If, after moving to a devel subpackage everything that could be moved, you have nothing left in the base package then you can probably keep everything in the base package.
Otherwise a split desirable.

Would it make sense to install the base package only and leave the devel subpackage in `$PACKAGES_PATH` ? If yes then I'd say the split is desirable.

Reply:
Nothing left in the base